### DISCUSSION TITLE
netdata for termux

### DISCUSSION THREAD

**Discussion by DocMAX** [termux/termux-app#3781] [General]
can netdata be ported to termux please? i try to compile but it fails. thanks.

**Comment by TomJo2000** [comment]
It would likely require substantial patches but could probably be done if there is demand for the package.
Feel free to open up [a package request](https://github.com/termux/termux-packages/issues/new?assignees=&labels=package+request&projects=&template=02-package-request.yml&title=%5BPackage%5D%3A+), and feel free to include your current findings.

**Comment by TomJo2000** [comment]
Closing in favor of termux/termux-packages#18969, which is the more appropriate place for this.
